### PR TITLE
Added manjola

### PR DESCRIPTION
**Pull request by ricardobarantini**

### CODE REVIEW

**Comment by joaocunha** [comment]
Thanks for the contribution!

Do you mind making sure the list is ordered alphabetically and leaving the trailing new line?

File: names.json [hunk 1]
```diff
@@ -72,8 +71,10 @@
   "Pistola",
   "Pitoca",
   "Piu-piu",
+  "Piça",
   "Pomba",
   "Provolone",
+  "Pênis",
```
Comment (joaocunha) [inline names.json]: Unordered.

File: names.json [hunk 2]
```diff
@@ -88,6 +89,6 @@
   "Trozoba",
   "Vara",
   "Veiúdo",
-  "Vergalhão",
-  "Verga"
-]
+  "Verga",
+  "Vergalhão"
+]
```
Comment (joaocunha) [inline names.json]: New line.

**Comment by ricardobarantini** [comment]
I used Sublime Text sort line function, I think they have a problem with accentuation. Sorry.

**Comment by joaocunha** [comment]
@ricardobarantini my bad - apparently that's how the alphabetical sorting works when you have special characters like `ê`.

Just add the trailing line back and I'll merge, please!

**Comment by ricardobarantini** [comment]
Done. :heavy_check_mark:

**Comment by joaocunha** [comment]
@ricardobarantini :eggplant:! 

On our way to :100: :)

**Pull request merged**